lambdas/service/app.py

```diff
@@ -1320,26 +1333,56 @@ def manifest_route(*, fetch: bool, initiate: bool):
         },
         method_spec={
             'tags': ['Manifests'],
+            'deprecated': curl,
             'summary':
                 (
                     'Initiate the preparation of a manifest'
                     if initiate else
                     'Determine status of a manifest preparation job'
                 ) + (
                     ' via XHR' if fetch else ''
+                ) + (
+                    ' as an alternative to PUT for curl users' if curl else ''
                 ),
-            'description': fd('''
-                Create a manifest preparation job, returning either
+            'description': (
+                fd('''
+                    Create a manifest preparation job, returning either
 
-                - a 301 redirect to the URL of the status of that job or
+                    - a 301 redirect to the URL of the status of that job or
 
-                - a 302 redirect to the URL of an already prepared manifest.
-
-                This endpoint is not suitable for interactive use via the
-                Swagger UI. Please use [PUT /fetch/manifest/files][1] instead.
+                    - a 302 redirect to the URL of an already prepared manifest.
+                ''')
+                + iif(not curl, fd(f'''
+                    This endpoint is not suitable for interactive use via the
+                    Swagger UI. Please use [{method} /fetch/manifest/files][1]
+                    instead.
 
-                [1]: #operations-Manifests-put_fetch_manifest_files
-            ''') + parameter_hoisting_note('PUT', '/manifest/files', 'PUT')
+                    [1]: #operations-Manifests-{method.lower()}_fetch_manifest_files
+                '''))
+                + parameter_hoisting_note(method, '/manifest/files', method)
+                + iif(curl, fd(f'''
+                    Requests to this endpoint are idempotent, so PUT would be
+                    the more standards-compliant method to use. POST is offered
+                    as a convenience for `curl` users, exploiting the fact that
+                    `curl` drops to GET when following a redirect in response to
+                    a POST, but not a PUT request. This is the only reason for
+                    the deprecation of this endpoint and there are currently no
+                    plans to remove it.
+
+                    To use this endpoint with `curl`, pass the `--location` and
+                    the `--data` options, as in `curl --data "" --location
```

Comment:
The example command line should on a separate paragraph, preferably in a pre-formatted text block.